packages/core/src/App/Containers/Layout/header/header.jsx

```diff
@@ -10,8 +10,7 @@ import { useLocation } from 'react-router-dom';
 const Header = ({ is_logged_in, is_pre_appstore }) => {
     const { is_appstore } = React.useContext(PlatformContext);
     const { pathname } = useLocation();
-    const trading_hub_routes =
-        pathname === routes.traders_hub || pathname.startsWith(routes.cashier) || pathname.startsWith(routes.account);
+    const trading_hub_routes = pathname === routes.traders_hub || pathname.startsWith(routes.account);
```

Comment:
@aizad-deriv What we should do is to show just accountSwitcher part to the header of `cashier` page and all the other sections should follow the header of `trader's hub`(`reposrts` in not visible, Exit trader's hub button is visible, ...)

Reply:
so removing cashier from `trading_hub_routes` is not correct solution